src/utils.cxx

```diff
@@ -211,16 +211,23 @@ void lower(std::string& str) {
 }
 
 std::string quoteAsNeeded(std::string& str) {
-    if (str.find_first_of(" &<>|()") != std::string::npos) {
-        // There are spaces or special characters in string, quote it
-        return "\"" + str + "\"";
-    }
+    // Note: the ordering if these two conditionals is important
+    // If the second conditional is executed first, the first
+    // will always be true as the second injects the string
+    // on which the first is conditioned
+    // Basically: If we find escaped strings: escape em again
```

Comment:
I think this just refers to all `\`, e.g. appearing as a path character above (so not necessarily escapes).